pkg/apis/application/v1alpha1/types.go

```diff
@@ -121,6 +121,23 @@ func (k *ApplicationSourceKustomize) IsZero() bool {
 	return k.NamePrefix == ""
 }
 
+// ApplicationSourceJsonnet holds jsonnet specific options
+type JsonnetVar struct {
+	Key  string `json:"key" protobuf:"bytes,1,opt,name=key"`
+	Val  string `json:"val" protobuf:"bytes,2,opt,name=val"`
+	Code bool   `json:"code,omitempty" protobuf:"bytes,3,opt,name=code"`
+}
+
+type ApplicationSourceJsonnet struct {
+	// Environment is a ksonnet application environment name
+	ExtVars []JsonnetVar `json:"extVars,omitempty" protobuf:"bytes,1,opt,name=extVars"`
+	Tlas    []JsonnetVar `json:"tlas,omitempty" protobuf:"bytes,2,opt,name=tlas"`
```

Comment:
I believe golang styling standard suggests we name this acronym TLAs

Reply:
kk